src/ert/storage/local_ensemble.py

```diff
@@ -357,15 +353,24 @@ def load_gen_data_as_df(
         return pd.concat(dfs)
 
     def load_field(self, key: str, realizations: List[int]) -> npt.NDArray[np.double]:
-        result = []
+        result: Optional[npt.NDArray[np.double]] = None
         for realization in realizations:
             input_path = self.mount_point / f"realization-{realization}"
             if not input_path.exists():
                 raise KeyError(f"Unable to load FIELD for key: {key}")
```

Comment:
Would it be better if the error message also mentioned the realization-number?
Something like this perhaps:

f"Unable to load FIELD for key {key} in realization {realization}."